cue.c

```diff
@@ -254,7 +257,6 @@ static int cue_read_comments(struct input_plugin_data *ip_data, struct keyval **
 	Track *t;
 	Rem *track_rem;
 	Cdtext *track_cdtext;
-	char *val;
```

Comment:
I've revert back to c90. Thanks. 

The reason we need to have another variable (path) in addition to child_filename is because free() takes a non-const pointer.